tools/deployment/osqueryctl

```diff
@@ -33,6 +37,8 @@ platform OS
 
 if [ $OS = "darwin" ]; then
   REAL_CONFIG_PATH="/var/osquery/osquery.conf"
+  FLAGS_FILE_PATH="/var/osquery/osquery.flags"
+  SECRETS_FILE_PATH="/var/osquery/osquery.secret"
```

Comment:
Ahhh sorry, I was trying to be complete and making sure that stuff was captured in case it was useful later. I went back and forth about checking for the secrets path. I can think of pros and cons. What's your take?